packages/forms-web-app/src/controllers/projects/documents.js

```diff
@@ -41,14 +43,55 @@ function renderData(
 	const respData = response.data;
 	const { documents, filters } = respData;
 	const { stageFilters, typeFilters } = filters;
-	logger.debug(`Document data received:  ${JSON.stringify(documents)} `);
 	const paginationData = getPaginationData(respData);
 	const pageOptions = calculatePageOptions(paginationData);
 	const modifiedStageFilters = [];
 	const top5TypeFilters = [];
-	const documentExaminationLibraryId = 'examination library';
 	let documentExaminationLibraryIndex = null;
 
+	if (documents.length && examinationLibraryResponse) {
+		for (var i = 0; i < documents.length; i++) {
+			const document = documents[i];
+			const documentType = typeof document.type === 'string' ? document.type.toLowerCase() : '';
+			if (documentType === documentExaminationLibraryId) {
+				documentExaminationLibraryIndex = i;
+				break;
+			}
+		}
+
+		if (documentExaminationLibraryIndex !== null) {
+			const documentElement = documents.splice(documentExaminationLibraryIndex, 1)[0];
+			documents.splice(0, 0, documentElement);
+		} else {
+			const examinationLibraryDocuments = response?.data?.documents;
+
+			if (
+				examinationLibraryDocuments &&
+				Array.isArray(examinationLibraryDocuments) &&
+				examinationLibraryDocuments.length
+			) {
+				const findExaminationLibraryDocumentType = examinationLibraryDocuments.find(
+					(examinationLibraryDocument) => {
+						const examinationLibraryDocumentType =
+							typeof examinationLibraryDocument.type === 'string'
+								? examinationLibraryDocument.type.toLowerCase()
+								: '';
+						return examinationLibraryDocumentType === documentExaminationLibraryId;
+					}
+				);
+
+				if (findExaminationLibraryDocumentType) {
+					documents.unshift(findExaminationLibraryDocumentType);
+				}
+			}
+		}
+	}
+
+	if (documents.length)
```

Comment:
Sorry to be pendantic but I would add a curly for this if even knowing it's not required... (optional)